dialogue-target/src/main/java/com/palantir/dialogue/HttpMethod.java

```diff
@@ -24,4 +24,5 @@
     PATCH,
     POST,
     PUT,
+    OPTIONS
```

Comment:
Alphabetize!

nit: last element should have a trailing comma so we get a nice one-line diff next time as well :-)
```suggestion
    OPTIONS,
```